src/epub/text/chapter-14.xhtml

```diff
@@ -39,8 +39,7 @@
 			<p>Now there had been no cloakroom ticket found in Sorrell’s possession.</p>
 			<p>“Did you see the ticket?”</p>
-			<p>“No; he only offered to give it to me.”</p>
-			<p>“Next morning I was late because I wasn’t used to doing for myself, and I had to make my own breakfast and tidy up, but I didn’t hurry because I had no job. I was hoping to get a clerk’s place when the ‘flat’ started. It was nearly twelve when I went out, and I wasn’t thinking of anything but Bert. I was so fed up with the way we’d parted and the fool I’d made of myself that I went to a post office and sent a wire to Bert addressed to the <i epub:type="se:name.vessel.ship">Queen of Arabia</i>, saying, ‘Sorry.⁠—<b>Jerry.</b>’ ”</p>
+			<p>“No; he only offered to give it to me. Next morning I was late because I wasn’t used to doing for myself, and I had to make my own breakfast and tidy up, but I didn’t hurry because I had no job. I was hoping to get a clerk’s place when the ‘flat’ started. It was nearly twelve when I went out, and I wasn’t thinking of anything but Bert. I was so fed up with the way we’d parted and the fool I’d made of myself that I went to a post office and sent a wire to Bert addressed to the <i epub:type="se:name.vessel.ship">Queen of Arabia</i>, saying, ‘Sorry.⁠—<b>Jerry.</b>’ ”</p>
```

Comment:
Page scans have this as a separate paragraph, but line 41 has no closing quote because it's the same speaker. Can you make this change?

Reply:
Lines 42-43 (prior to my edit) are a different speaker from line 41, as is apparent when reading the chapter. Do you mean that line 42 should be: `<p>“No; he only offered to give it to me.</p>` (without an ending quotation mark)?

Where can I find the scans you refer to? Is it possible there's an editing mistake in the original book?

Reply:
The page scans are linked in the ebook's download page, as well as in the ebook's imprint and colophon. Here is the page: 
![Screenshot 2025-02-16 at 17-06-52 The man in the queue Tey Josephine 1896 or 1897-1952 author Free Download Borrow and Streaming Internet Archive](https://github.com/user-attachments/assets/6bebdda2-6848-49cb-8a29-7d975389bc37)

Reply:
Change made. You can probably do a squash commit on this PR now.